.github/workflows/ci.yml

```diff
@@ -145,3 +147,9 @@ jobs:
           # message to the pull request title
           # https://github.com/cypress-io/github-action/issues/124#issuecomment-653180260
           COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
+
+          # Percy config
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+          PERCY_PARALLEL_TOTAL: 3
```

Comment:
```suggestion
```

We could try removing this as well: [^docs]

> `PERCY_PARALLEL_TOTAL`: The total number of parallel builds to expect.
> - In most setups this is automatically discovered and does not need to be set. See Parallel test suites for more info.

[^docs]: https://docs.percy.io/docs/environment-variables#optional

Reply:
Let's try this one, although I'm not sure how Percy can derive this one since it depends on the amount of containers defined in the job.